packages/devtools/lib/src/inspector/inspector_service.dart

```diff
@@ -865,27 +865,6 @@ class ObjectGroup {
   }
 }
 
-// TODO(jacobr): can we get the host OS from VMService.
```

Comment:
great to see this removed. it is broken in devtools anyway. :)

Reply:
:-)